plugin/src/main/java/com/iridium/iridiumskyblock/commands/BankCommand.java

```diff
@@ -46,27 +49,32 @@ public void execute(CommandSender sender, String[] args) {
                     User user = IridiumSkyblock.getInstance().getUserManager().getUser(player);
                     Optional<Island> island = user.getIsland();
                     if (island.isPresent()) {
-                        Optional<BankItem> bankItem =
-                                IridiumSkyblock.getInstance().getBankItemList().stream().filter(item -> item.getName().equalsIgnoreCase(args[3])).findFirst();
+                        Optional<BankItem> bankItem = IridiumSkyblock.getInstance().getBankItemList().stream().filter(item -> item.getName().equalsIgnoreCase(args[3])).findFirst();
                         if (bankItem.isPresent()) {
                             try {
-                                IslandBank islandBank =
-                                        IridiumSkyblock.getInstance().getIslandManager().getIslandBank(island.get(), bankItem.get());
+                                IslandBank islandBank = IridiumSkyblock.getInstance().getIslandManager().getIslandBank(island.get(), bankItem.get());
                                 islandBank.setNumber(islandBank.getNumber() + Double.parseDouble(args[4]));
-                                sender.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().gaveBank.replace("%player%", player.getName()).replace("%amount%", args[4]).replace("%item%", bankItem.get().getName()).replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)));
+                                IridiumSkyblock.getInstance().getDatabaseManager().getIslandTransactionTableManager().getEntries().add(
```

Comment:
Cant we add this to BankItem#Withdraw?

Reply:
Oh i didn't known deleted sendMessage line sorry
also we can't cause we should known command executed by admin or console

Reply:
We could just add a method parameter for that